getting-started-vue/11.md

```diff
@@ -0,0 +1,174 @@
+## Lesson 11. A few masterful touches on iOS
+
+In this lesson, you'll learn how to perform platform checks and tie platform-specific logic and design to them.
+
+* You will improve the look and feel of the `<ActionBar>`.
+* You will tweak the keyboard on iOS.
+* You will add iOS-specific icons in the `<TabView>`.
+
+### Step 1. Access the `platform` and the `frame` modules
+
+The core `platform` module lets you get device-specific information such as device model, type, OS and OS version, language, manufacturer, and screen information. It lets you perform OS checks and implement platform- and device-specific functionality that depends on the these checks.
+
+You will later use the `frame` to access the iOS native component behind the `<ActionBar>` and change its style and color.
+
+To tap into the powers of these modules, you need to require them explicitly in your code.
+
+<hr data-action="start" />
+
+#### Action
+
+* **a.** In `HelloWorld.vue` > `<script>`, require the `platform` and the `frame` module. Place these statements at the top of the block before the `export` statement.
+
+```JavaScript
+const platform = require("tns-core-modules/platform");
+const frame = require("tns-core-modules/ui/frame");
+```
+
+<hr data-action="end" />
+
+### Step 2. Improve the looks of the `<ActionBar>` on iOS
+
+Note how the `<ActionBar>` at the top of the screen combines a dark label on a dark background for iOS. You can change this with a simple platform-specific tweak. Use the `platform.isIOS` check to run the configuration change only on iOS. Next, apply a specific iOS status bar setting.
```

Comment:
I think you mean status bar and not action bar in the opening sentence of this paragraph.